### PR TITLE
Add the RCA2 cert and use it when making Drupal requests.

### PR DESCRIPTION
**Pull request by timcosgrove**
## Description
To fix https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12285

## Testing done & Screenshots
Run content-build locally; it passed including downloading files.

To make sure Tugboat is reached correctly:
```
yarn build --pull-drupal
```

To make sure Staging continues to be reached correctly:
```
yarn build --pull-drupal --drupal-address=https://staging.cms.va.gov
```

### CODE REVIEW

File: src/site/stages/build/drupal/api.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ function getDrupalClient(buildOptions, clientOptionsArg) {
         // addCAs() is here because VA uses self-signed certificates with a
         // non-globally trusted Root Certificate Authority and we need to
         // tell our code to trust it, otherwise we get self-signed certificate errors.
-        syswidecas.addCAs('certs/VA-Internal-S2-RCA1-v1.pem');
```
Comment (ndouglas) [inline src/site/stages/build/drupal/api.js:72]: Not sure it's good to remove old cert.

Comment (timcosgrove) [reply src/site/stages/build/drupal/api.js:72]: revised

**Review by ndouglas — APPROVED** [review]
LGTM 

(:trollface:)

**Review by jtmst — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**